document/src/vespa/document/select/parser.cpp

```diff
@@ -8,7 +9,18 @@
 
 namespace document::select {
 
+namespace {
+
+void verify_expression_not_too_large(const std::string& expr) {
+    if (expr.size() > ParserLimits::MaxSelectionByteSize) {
+        throw ParsingFailedException("expression is too large to be parsed");
```

Comment:
Consider adding the size and limit to the exception message.

Reply:
Done